### DISCUSSION TITLE
env: passed to pre?

### DISCUSSION THREAD

**Discussion by hongkongkiwi** [finit-project/finit#240] [Q&A] [answered]
I couldn't find my answer in the docs.

Are env:.... variables passed to the pre/post scripts also? I'm interested in having one place for the vars and using them both in pre: and also running the app.

**Comment by troglobit** [comment, marked as answer]
Good catch, you're right the env: variables should be passed to the pre/post scripts, but they don't right now.  I'll convert this to an issue and fix it before the 4.3 release!
